environment.js

```diff
@@ -0,0 +1,26 @@
+var path = require('path');
+var fs = require('fs');
+var mkdirp = require('mkdirp');
+var glob = require('glob');
+
+export.ensureDirectory = function(directoryName) {
+  if (!fs.existsSync(directoryName)) {
+    mkdirp.sync(directoryName);
+  }
+};
+
+export.loadSpecifications = function(mocha, directory) {
+   function addSpecs(error, files) {
```

Comment:
Personally I don't like hoisting but whatever your opinion it is a language feature... Let' not get bogged down in the philosophical: merge, merge, merge!!